chart/kubeapps/templates/kubeappsapis/serviceaccount.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.featureFlags.kubeappsAPIsServer }}
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Why do we need a SA at the moment for this service?

Reply:
Yep, as you saw, for the dev-only RBAC until I add the user auth.